src/home_work3/api/suggestionRoutes.js

```diff
@@ -0,0 +1,30 @@
+let express = require("express");
+let router = express.Router();
+const Joi = require("joi");
+const validator = require("express-joi-validation").createValidator({});
+
+let userModel = require("../models/UserModel");
```

Comment:
Consider defaulting to usage of "const" wherever possible.

Reply:
Resolved as per the comment